tests/integration/ha/test_network_cut.py

```diff
@@ -11,10 +11,17 @@
 import yaml
 from pytest_operator.plugin import OpsTest
 
-from ..helpers import access_all_dashboards, get_address, get_leader_name
+from ..helpers import (
+    access_all_dashboards,
+    all_dashboards_unavailable,
+    get_address,
+    get_leader_name,
+)
 
 logger = logging.getLogger(__name__)
 
+logging.basicConfig(format="%(asctime)s %(levelname)-8s %(message)s", datefmt="%Y-%m-%d %H:%M:%S")
+
```

Comment:
What's the reason for this? I assume this should be handled by the test runner (in this case, tox), no?

Reply:
Ah, it was a naive attempt to get timestamps on the CI messages. Didn't work, removing.

Reply:
Could we remove?

Reply:
Yes, I'm getting back to do all changes at once, when all review is collected and discussions are finalized.
(In attempt to reduce context switch.)

Reply:
Done.